dashboard/src/components/InstanceListView/index.tsx

```diff
@@ -20,7 +21,7 @@ import { InstanceCardList } from "./InstanceCardList";
 export interface InstanceListViewProps {
   brokers: IServiceBroker[];
   classes: IClusterServiceClass[];
-  error: Error;
+  error: Error | undefined;
```

Comment:
can this be `error?: Error`? Sometimes Typescript complains about this though.

Reply:
I couldn't but I can omit the `error` parameter so I don't need that change.